src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -35,7 +35,7 @@
             v-model="notation"
```

Comment:
Re: line 30

Was there a UX requirement to change "Detail" to "details"?

Reply:
@ketaki-deodhar ^^

Reply:
@severinbeauvais, not in this ticket